pygmt/src/pygmtlogo.py

```diff
@@ -0,0 +1,311 @@
+"""
+pygmtlogo - Create and plot the PyGMT logo.
+The design of the logo is kindly provided by `@sfrooti <https://github.com/sfrooti>`_
+and consists of a visual and the wordmark "PyGMT".
+"""
+
+from pathlib import Path
+
+import numpy as np
+import pygmt
+
+
+def create_logo(color=True, theme="light", shape="circle", wordmark=True):  # noqa: PLR0915
```

Comment:
Reorder the parameters in descending order of their expected usage frequency.
```suggestion
def create_logo(
        shape : Literal["circle", "hexagon"] = "circle",
        theme : Literal["light", "dark"] = "light",
        wordmark : Literal["horizontal", "vertical"] | bool = True,
        color : bool = True,
    ):  # noqa: PLR0915
```